<ui version="4.0" >
 <class>settingsDlg</class>
 <widget class="QDialog" name="settingsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>You</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="player1Name" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maxLength" >
           <number>32</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Match settings</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Match length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="matchLength" >
          <property name="toolTip" >
           <string>Choose the length of the match in points</string>
          </property>
          <property name="whatsThis" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">In Greece, tables games are called &lt;span style=" font-weight:600;">tavli&lt;/span> (related to the word &lt;span style=" font-style:italic;">tavla&lt;/span>, meaning "board" or "table", and &lt;a href="http://en.wikipedia.org/wiki/Cognate">&lt;span style=" text-decoration: underline; color:#0000ff;">cognate&lt;/span>&lt;/a> to the Latin &lt;span style=" font-style:italic;">tabula&lt;/span>). There are three major variants, &lt;span style=" font-weight:600; font-style:italic;">portes&lt;/span>, &lt;span style=" font-weight:600; font-style:italic;">plakoto&lt;/span>, and &lt;span style=" font-weight:600; font-style:italic;">fevga&lt;/span>. Portes resembles &lt;a href="http://en.wikipedia.org/wiki/Backgammon">&lt;span style=" text-decoration: underline; color:#0000ff;">backgammon&lt;/span>&lt;/a>, with minor variations: there is no doubling cube, and a backgammon counts only as a gammon (called &lt;span style=" font-style:italic;">diplo&lt;/span>, Greek for &lt;span style=" font-style:italic;">double&lt;/span>). Plakoto is very similar to mahbusa or &lt;a href="http://en.wikipedia.org/wiki/Tapa_%28game%29">&lt;span style=" text-decoration: underline; color:#0000ff;">tapa&lt;/span>&lt;/a>, while fevga is similar to narde or the Turkish variant &lt;span style=" font-weight:600; font-style:italic;">moultezim&lt;/span>. The three are normally played consecutively, in three-, five- or seven-point matches.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; font-weight:600;">Source: Wikipedia&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="specialValueText" >
           <string>1 point</string>
          </property>
          <property name="suffix" >
           <string> points</string>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>7</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="checkBox_portes" >
          <property name="text" >
           <string>Portes</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_plakoto" >
          <property name="text" >
           <string>Plakoto</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_fevga" >
          <property name="text" >
           <string>Fevga</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Opponent</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="radioButton" >
          <property name="text" >
           <string>Network</string>
          </property>
          <property name="icon" >
           <iconset resource="tavli.qrc" >:/images/network.png</iconset>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remoteIP" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Computer</string>
        </property>
        <property name="icon" >
         <iconset resource="tavli.qrc" >:/images/server.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>211</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tavli.qrc" />
 </resources>
 <connections/>
</ui>
